modules/manually-configure-iam-nutanix.adoc

```diff
@@ -78,21 +68,59 @@ $ ccoctl nutanix create-shared-secrets \
       namespace: openshift-machine-api
 ----
 
-. Create the installation manifests:
+. Use the `ccoctl` tool to process all of the `CredentialsRequest` objects in the `credrequests` directory by running the following command:
++
+[source,terminal]
+----
+$ ccoctl nutanix create-shared-secrets \
+--credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+--output-dir=<ccoctl_output_dir> \ <2>
+--credentials-source-filepath=<path_to_credentials_file> <3>
+----
++
+<1> Specify the path to the directory that contains the files for the component `CredentialsRequests` objects.
+<2> Specify the directory that contains the files of the component credentials secrets, under the `manifests` directory. By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag.
+<3> Optional: Specify the directory that contains the credentials data YAML file. By default, `ccoctl` expects this file to be in `<home_directory>/.nutanix/credentials`. To specify a different directory, use the `--credentials-source-filepath` flag.
+
+. Edit the `install-config.yaml` configuration file so that the `credentialsMode` parameter is set to `Manual`.
++
+.Example `install-config.yaml` configuration file
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Manual <1>
+...
+----
+<1> Add this line to set the `credentialsMode` parameter to `Manual`.
+
+. Create the installation manifests by running the following command:
 +
 [source,terminal]
 ----
 $ openshift-install create manifests --dir <installation_directory><1>
 ----
-<1> `<installation_directory>` specifies the name of the directory that contains the `install-config.yaml` file for your cluster.
+<1> Specify the path to the directory that contains the `install-config.yaml` file for your cluster.
 
-. Copy the generated credential files to the target manifests directory:
+. Copy the generated credential files to the target manifests directory by running the following command:
 +
 [source,terminal]
 ----
-$ cp <output_dir>/manifests/*credentials.yaml ./<installation_directory>/manifests
+$ cp <ccoctl_output_dir>/manifests/*credentials.yaml ./<installation_directory>/manifests
 ----
 
 .Verification
 
-* Ensure that the appropriate secrets were generated in your cluster's `manifests` directory.
+* Ensure that the appropriate secrets exist in the `manifests` directory.
++
+[source,terminal]
+----
+$ ls ./<installation_directory>/manifests
+----
++
+.Example output
++
+[source,terminal]
+----
+#Is there a sample output here?#
```

Comment:
Note: would like an output here but it does not need to hold this PR up. Can omit if needed, but hopefully will be available before docs freeze.

Reply:
here is sample output after copy command

```
[akhilrane@localhost nutanix]$ cp manifests_ccoctl/manifests/*credentials.yaml ./manifests-installer/manifests
[akhilrane@localhost nutanix]$ ls manifests-installer/manifests/
cluster-config.yaml                   cluster-network-02-config.yml    kube-system-configmap-root-ca.yaml
cluster-dns-02-config.yml             cluster-proxy-01-config.yaml     machine-config-server-tls-secret.yaml
cluster-infrastructure-02-config.yml  cluster-scheduler-02-config.yml  openshift-config-secret-pull-secret.yaml
cluster-ingress-02-config.yml         cvo-overrides.yaml               openshift-machine-api-nutanix-credentials-credentials.yaml
cluster-network-01-crd.yml            kube-cloud-config.yaml
[akhilrane@localhost nutanix]$ 

```